docs: Fix incorrect parentheses (#9629)
jekyll/jekyll: docs/_docs/step-by-step/07-assets.md

The empty front matter at the top tells Jekyll it needs to process the file. The
`@import "main"` tells Sass to look for a file called `main.scss` in the sass

> directory `_sass/` by default (which you already created directly under the root folder of your website).

Reviewer: I prefer the following though:

```suggestion
directory (`_sass/` by default) you already created at the root of your working directory earlier.
```
